DiamondLightSource/python-artemis
# Mxgda 3733: Minimal gridscan plan

Fixes https://jira.diamond.ac.uk/browse/MXGDA-3733 and https://jira.diamond.ac.uk/browse/MXGDA-3765.

Note, is branched from https://github.com/DiamondLightSource/python-artemis/pull/9 so merge that first.

To test:
* Cherrypick the code in https://github.com/DiamondLightSource/python-artemis/tree/test_minimal_scan (this is required due to https://github.com/DiamondLightSource/python-artemis/issues/13 and https://github.com/DiamondLightSource/python-artemis/issues/11)
* Run this code i.e. `pipenv run python src/artemis/fast_grid_scan_plan.py`
* Confirm that the coe runs without any errors and gives a progress bar of what it's doing
* Confirm logs are produced in `/tmp/bluesky.log` and `/tmp/ophyd.log`
